Samurai/Program.cs

```diff
@@ -1,22 +1,18 @@
 ﻿using System;
-using System.Collections.Generic;
 using System.IO;
-using System.Linq;
 using System.Net.Http;
 using System.Threading;
 using System.Threading.Tasks;
+using AcidChicken.Samurai.Models;
```

Comment:
All of the local namespace reference instructions have to be included in the local namespace block.